db/schema.rb

```diff
@@ -1061,6 +1061,19 @@
     t.index ["api_name", "consumer_id", "api_guid"], name: "index_webhooks_subscription", unique: true
   end
 
+  create_table "virtual_agent_user_access_records", force: :cascade do |t|
+    t.string "action_type", null: false
+    t.string "first_name"
+    t.string "last_name"
+    t.string "ssn", null: false
+    t.string "icn",  null: false
+    t.datetime "created_at",  null: false
+
+    t.index [ "ssan" ], unique: false
+    t.index [ "icn" ],  unique: false
+    t.index [ "created_at" ], unique: false
```

Comment:
It's Rails' best practice to also add an `updated_at` timestamp. You can simply use `t.timestamps` in your migration.